components/Vote/201912/DiscussionPage.js

```diff
@@ -0,0 +1,86 @@
+import React, { Fragment } from 'react'
+import Frame from '../../../components/Frame'
+import Discussion from '../../../components/Discussion/Discussion'
+import { withRouter } from 'next/router'
+import { mediaQueries } from '@project-r/styleguide'
+import { css } from 'glamor'
+import { compose, graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import { VOTING_COOP_201912_REPORT_SLUG } from '../constants'
+import voteT from '../voteT'
+import { Body, Section, Title } from '../text'
+import Loader from '../../Loader'
+import md from 'markdown-in-js'
+import { mdComponents } from '../text'
+
+const styles = {
+  discussion: css({
+    margin: '0 0 20px 0',
+    [mediaQueries.lUp]: {
+      margin: '30px 0'
+    }
+  })
+}
+
+const DISCUSSION_SLUG = VOTING_COOP_201912_REPORT_SLUG
+
+const DiscussionPage = ({ router, data, vt }) => {
+  const meta = {
+    title: vt('info/title'),
+    description: vt('info/description')
+  }
+
+  return (
+    <Frame meta={meta}>
+      <Loader
+        loading={data.loading}
+        error={data.error}
+        render={() => {
+          const discussionId =
+            data[DISCUSSION_SLUG] && data[DISCUSSION_SLUG].discussion.id
+          return (
+            <Fragment>
+              <Title>{vt('vote/201912/discussion/title')}</Title>
+              {md(mdComponents)`
+Während der [Abstimmung](/vote/dez19) vom 13. bis zum 23. Dezember 2019 möchten wir mit Ihnen diskutieren. Wenn Sie etwas genauer wissen möchten, Lob oder Kritik für uns haben: Wir freuen uns auf Ihren Beitrag.
+
+ - Haben Sie Verständnisfragen?
```

Comment:
A FYI: `markdown-in-js` will render no p tag for compact lists and p tags for wide lists.

<img width="1371" alt="Screenshot 2019-12-11 at 16 34 14" src="https://user-images.githubusercontent.com/410211/70635549-69f89f80-1c34-11ea-976e-d5fb05c7cbc7.png">

(if you experience inconsistent font size, had an issue with that for cockpit)